platform/viewer/src/components/ViewportGrid.tsx

```diff
@@ -4,6 +4,7 @@ import { ViewportGrid, ViewportPane, useViewportGrid } from '@ohif/ui';
 import { utils } from '@ohif/core';
 import EmptyViewport from './EmptyViewport';
 import classNames from 'classnames';
+import IDisplaySet from '@ohif/core';
```

Comment:
pretty sure the deafult export of ohif/core is not Idiplayset

Reply:
🤣